client/js/screens/Home/Home.js

```diff
@@ -32,7 +32,7 @@ const MOCK_DATA = [
 const Home = ({navigation}) => {
   return (
     <>
-      <View style={styles.carouselContainer}>
+      <View height={'22%'} style={styles.carouselContainer}>
```

Comment:
Please remove all the inline styles and move them to styles.js file.

Reply:
Done